src/rpc/RPC.cpp

```diff
@@ -346,6 +346,8 @@ buildResponse(Context const& ctx)
     if (ctx.method == "ping")
         return boost::json::object{};
 
+    // FIXME: probably not a real usecase that we should support
```

Comment:
👍  I don't think we should regulate traffic here too. But understand we still need to keep this interface for now. If it will return hardcode false value, we can remove maxReadRequestsOutstanding_ from new backend then.

Reply:
Actually, this comment is a lie as it is now - i have re-added this functionality in the new backend after i had it return `false` and forgot to update this comment. While i agree we should eventually move this somewhere else, let's keep the logic as close to what it was before as possible so that other parts of clio don't fall apart.